source-code/sdk-js/src/config/index.ts

```diff
@@ -0,0 +1,47 @@
+import {
+	Config as InlangConfig,
+	EnvironmentFunctions,
+	initialize$import,
+} from "@inlang/core/config"
+
+// this is a copy from `source-code/website/src/pages/editor/@host/@owner/@repository/State.tsx`
+// TODO: move functionality into `core/config`
```

Comment:
We will need this functionality a few times. @samuelstroschein should we create a reusable function?

Reply:
Likely not. If you are referring to `readInlangConfig`, then you shouldn't use the function from the browser. 

The browser implementation uses a virtual filesystem which complicates things. You want to use `node:fs/promises` for the SDK. After all, the resources should be read from the local repository. Furthermore, you can directly import the inlang.config.js file in a node runtime. There is no need to import the module with mimeType. 

Taken from the [ide extension](https://github.com/inlang/inlang/blob/main/source-code/ide-extension/src/main.ts#L63-L67):

```ts

const $import = initialize$import({ fs, fetch })
// you just need to know the path of the config. the config must be at the root of a repo for now.
const module = await import("./inlang.config.js")
const config = await module.defineConfig({ $fs: fs, $import })

```